components/browse/HeroHeader.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <div class="container hero-wrapper">
+    <b-jumbotron
+      :style="jumbotronStyle"
+      fluid
+      class="browse-jumbotron"
+      text-variant="white"
+      data-qa="hero banner"
+      @click="toggleCite"
```

Comment:
Is this here with a reason? 
As it opens the attribution no matter where you click on the hero banner.

Reply:
this was added back in summer as one of the acceptance criteria for the new citations, clicking outside of the citation should close it. It does of course have the reverse effect too of opening the citation if it's closed

Reply:
Ah yes, now I see that it is working similarly on exhibitions/blogs. 
What I find slightly annoying is that if you click the CTA button, you briefly see the attribution before going to the link.

Reply:
yeah that is annoying, we could change it to something like `toggleCite ? !toggleCite : null`